### ISSUE TITLE
Safari JavaScript compat data: provide real version numbers

### ISSUE THREAD

**Issue by vinyldarkscratch** [mdn/browser-compat-data#4948]
Part of #4771. Below are JavaScript features that are set to `true` or `null` for Safari (macOS edition). The goal of this issue is to find the actual version numbers these features were implemented, if at all.
Total amount of features: 436

_Note: Until Chrome 27, Chrome used the same engine as Safari (WebKit).  As a result, you may be able to add Safari data for any features implemented in Chrome 1-27 by comparing the WebKit engine versions described in [browsers/chrome.json](https://github.com/mdn/browser-compat-data/blob/master/browsers/chrome.json) and [browser/safari.json](https://github.com/mdn/browser-compat-data/blob/master/browsers/safari.json).  Additionally, Chrome 1 used a WebKit version just before Safari 4, so any features implemented in Safari 1-3.2 may be set to Chrome 1.  Chrome 27 and Safari 6 were the last versions that maintained parity before Chrome forked WebKit into the Blink engine._

_Another note: I have VMs for Safari 1 through 7 should anyone interested in helping need copies.  Please [email me](https://github.com/vinyldarkscratch) with the subject “MDN Safari VMs” and I’ll send a download link!  For Safari 8+, we recommend a program like SauceLabs or BrowserStack; alternatively I can configure VMs per request._

<details>
<summary>1: Array (#5249) ✅</summary>
<ul>
	<li>javascript.builtins.Array</li>
	<li>javascript.builtins.Array.concat</li>
	<li>javascript.builtins.Array.every</li>
	<li>javascript.builtins.Array.filter</li>
	<li>javascript.builtins.Array.forEach</li>
	<li>javascript.builtins.Array.indexOf</li>
	<li>javascript.builtins.Array.join</li>
	<li>javascript.builtins.Array.lastIndexOf</li>
	<li>javascript.builtins.Array.length</li>
	<li>javascript.builtins.Array.map</li>
	<li>javascript.builtins.Array.pop</li>
	<li>javascript.builtins.Array.prototype</li>
	<li>javascript.builtins.Array.push</li>
	<li>javascript.builtins.Array.reverse</li>
	<li>javascript.builtins.Array.shift</li>
	<li>javascript.builtins.Array.slice</li>
	<li>javascript.builtins.Array.some</li>
	<li>javascript.builtins.Array.sort</li>
	<li>javascript.builtins.Array.sort.stable</li>
	<li>javascript.builtins.Array.splice</li>
	<li>javascript.builtins.Array.toLocaleString</li>
	<li>javascript.builtins.Array.toString</li>
	<li>javascript.builtins.Array.unshift</li>
	<li>javascript.builtins.Array.@@iterator</li>
	<li>javascript.builtins.Array.@@species</li>
	<li>javascript.builtins.Array.@@unscopables</li>
	<li>javascript.builtins.ArrayBuffer.isView</li>
	<li>javascript.builtins.ArrayBuffer.@@species</li>
	<li>javascript.builtins.Float32Array.iterable_in_constructor</li>
	<li>javascript.builtins.Float64Array.iterable_in_constructor</li>
	<li>javascript.builtins.Int16Array.iterable_in_constructor</li>
	<li>javascript.builtins.Int32Array.iterable_in_constructor</li>
	<li>javascript.builtins.Int8Array.iterable_in_constructor</li>
	<li>javascript.builtins.SharedArrayBuffer</li>
	<li>javascript.builtins.SharedArrayBuffer.byteLength</li>
	<li>javascript.builtins.SharedArrayBuffer.prototype</li>
	<li>javascript.builtins.SharedArrayBuffer.sab_in_dataview</li>
	<li>javascript.builtins.SharedArrayBuffer.slice</li>
	<li>javascript.builtins.TypedArray.iterable_in_constructor</li>
	<li>javascript.builtins.TypedArray.slice</li>
	<li>javascript.builtins.TypedArray.sort</li>
	<li>javascript.builtins.TypedArray.@@iterator</li>
	<li>javascript.builtins.TypedArray.@@species</li>
	<li>javascript.builtins.Uint16Array.iterable_in_constructor</li>
	<li>javascript.builtins.Uint32Array.iterable_in_constructor</li>
	<li>javascript.builtins.Uint8Array.iterable_in_constructor</li>
	<li>javascript.builtins.Uint8ClampedArray.iterable_in_constructor</li>
</ul>
</details>

<details>
<summary>2: Date (#5302) ✅</summary>
<ul>
	<li>javascript.builtins.Date</li>
	<li>javascript.builtins.Date.UTC</li>
	<li>javascript.builtins.Date.getDate</li>
	<li>javascript.builtins.Date.getDay</li>
	<li>javascript.builtins.Date.getFullYear</li>
	<li>javascript.builtins.Date.getHours</li>
	<li>javascript.builtins.Date.getMilliseconds</li>
	<li>javascript.builtins.Date.getMinutes</li>
	<li>javascript.builtins.Date.getMonth</li>
	<li>javascript.builtins.Date.getSeconds</li>
	<li>javascript.builtins.Date.getTime</li>
	<li>javascript.builtins.Date.getTimezoneOffset</li>
	<li>javascript.builtins.Date.getUTCDate</li>
	<li>javascript.builtins.Date.getUTCDay</li>
	<li>javascript.builtins.Date.getUTCFullYear</li>
	<li>javascript.builtins.Date.getUTCHours</li>
	<li>javascript.builtins.Date.getUTCMilliseconds</li>
	<li>javascript.builtins.Date.getUTCMinutes</li>
	<li>javascript.builtins.Date.getUTCMonth</li>
	<li>javascript.builtins.Date.getUTCSeconds</li>
	<li>javascript.builtins.Date.getYear</li>
	<li>javascript.builtins.Date.parse</li>
	<li>javascript.builtins.Date.parse.iso_8601</li>
	<li>javascript.builtins.Date.prototype</li>
	<li>javascript.builtins.Date.setDate</li>
	<li>javascript.builtins.Date.setFullYear</li>
	<li>javascript.builtins.Date.setHours</li>
	<li>javascript.builtins.Date.setMilliseconds</li>
	<li>javascript.builtins.Date.setMinutes</li>
	<li>javascript.builtins.Date.setMonth</li>
	<li>javascript.builtins.Date.setSeconds</li>
	<li>javascript.builtins.Date.setTime</li>
	<li>javascript.builtins.Date.setUTCDate</li>
	<li>javascript.builtins.Date.setUTCFullYear</li>
	<li>javascript.builtins.Date.setUTCHours</li>
	<li>javascript.builtins.Date.setUTCMilliseconds</li>
	<li>javascript.builtins.Date.setUTCMinutes</li>
	<li>javascript.builtins.Date.setUTCMonth</li>
	<li>javascript.builtins.Date.setUTCSeconds</li>
	<li>javascript.builtins.Date.setYear</li>
	<li>javascript.builtins.Date.toDateString</li>
	<li>javascript.builtins.Date.toGMTString</li>
	<li>javascript.builtins.Date.toISOString</li>
	<li>javascript.builtins.Date.toJSON</li>
	<li>javascript.builtins.Date.toLocaleDateString</li>
	<li>javascript.builtins.Date.toLocaleDateString.iana_time_zone_names</li>
	<li>javascript.builtins.Date.toLocaleString</li>
	<li>javascript.builtins.Date.toLocaleString.iana_time_zone_names</li>
	<li>javascript.builtins.Date.toLocaleTimeString</li>
	<li>javascript.builtins.Date.toLocaleTimeString.iana_time_zone_names</li>
	<li>javascript.builtins.Date.toString</li>
	<li>javascript.builtins.Date.toTimeString</li>
	<li>javascript.builtins.Date.toUTCString</li>
	<li>javascript.builtins.Date.valueOf</li>
	<li>javascript.builtins.Date.@@toPrimitive</li>
</ul>
</details>

<details>
<summary>3: Error (#5315) ✅</summary>
<ul>
	<li>javascript.builtins.Error</li>
	<li>javascript.builtins.Error.message</li>
	<li>javascript.builtins.Error.name</li>
	<li>javascript.builtins.Error.prototype</li>
	<li>javascript.builtins.Error.toString</li>
	<li>javascript.builtins.EvalError</li>
	<li>javascript.builtins.RangeError</li>
	<li>javascript.builtins.ReferenceError</li>
	<li>javascript.builtins.SyntaxError</li>
	<li>javascript.builtins.TypeError</li>
	<li>javascript.builtins.URIError</li>
</ul>
</details>

<details>
<summary>4: Function (#5332) ✅</summary>
<ul>
	<li>javascript.builtins.AsyncFunction</li>
	<li>javascript.builtins.AsyncFunction.prototype</li>
	<li>javascript.builtins.Function</li>
	<li>javascript.builtins.Function.apply</li>
	<li>javascript.builtins.Function.apply.generic_arrays_as_arguments</li>
	<li>javascript.builtins.Function.arguments</li>
	<li>javascript.builtins.Function.call</li>
	<li>javascript.builtins.Function.caller</li>
	<li>javascript.builtins.Function.displayName</li>
	<li>javascript.builtins.Function.length</li>
	<li>javascript.builtins.Function.length.configurable_true</li>
	<li>javascript.builtins.Function.name</li>
	<li>javascript.builtins.Function.name.configurable_true</li>
	<li>javascript.builtins.Function.name.inferred_names</li>
	<li>javascript.builtins.Function.prototype</li>
	<li>javascript.builtins.Function.toString</li>
	<li>javascript.builtins.GeneratorFunction</li>
	<li>javascript.builtins.GeneratorFunction.prototype</li>
	<li>javascript.functions</li>
	<li>javascript.functions.arguments</li>
	<li>javascript.functions.arguments.callee</li>
	<li>javascript.functions.arguments.length</li>
	<li>javascript.functions.arrow_functions.trailing_comma</li>
	<li>javascript.functions.block_level_functions</li>
	<li>javascript.functions.default_parameters.destructured_parameter_with_default_value_assignment</li>
	<li>javascript.functions.rest_parameters.destructuring</li>
</ul>
</details>

<details>
<summary>5: Math (#5334) ✅</summary>
<ul>
	<li>javascript.builtins.Math.E</li>
	<li>javascript.builtins.Math.LN2</li>
	<li>javascript.builtins.Math.LN10</li>
	<li>javascript.builtins.Math.LOG2E</li>
	<li>javascript.builtins.Math.LOG10E</li>
	<li>javascript.builtins.Math.PI</li>
	<li>javascript.builtins.Math.SQRT1_2</li>
	<li>javascript.builtins.Math.SQRT2</li>
	<li>javascript.builtins.Math.abs</li>
	<li>javascript.builtins.Math.acos</li>
	<li>javascript.builtins.Math.asin</li>
	<li>javascript.builtins.Math.atan</li>
	<li>javascript.builtins.Math.atan2</li>
	<li>javascript.builtins.Math.ceil</li>
	<li>javascript.builtins.Math.clz32</li>
	<li>javascript.builtins.Math.cos</li>
	<li>javascript.builtins.Math.exp</li>
	<li>javascript.builtins.Math.floor</li>
	<li>javascript.builtins.Math.log</li>
	<li>javascript.builtins.Math.max</li>
	<li>javascript.builtins.Math.min</li>
	<li>javascript.builtins.Math.pow</li>
	<li>javascript.builtins.Math.random</li>
	<li>javascript.builtins.Math.round</li>
	<li>javascript.builtins.Math.sin</li>
	<li>javascript.builtins.Math.sqrt</li>
	<li>javascript.builtins.Math.tan</li>
</ul>
</details>

<details>
<summary>6: Number (#5394) ✅</summary>
<ul>
	<li>javascript.builtins.Number</li>
	<li>javascript.builtins.Number.MAX_VALUE</li>
	<li>javascript.builtins.Number.MIN_VALUE</li>
	<li>javascript.builtins.Number.NaN</li>
	<li>javascript.builtins.Number.NEGATIVE_INFINITY</li>
	<li>javascript.builtins.Number.POSITIVE_INFINITY</li>
	<li>javascript.builtins.Number.isInteger</li>
	<li>javascript.builtins.Number.prototype</li>
	<li>javascript.builtins.Number.toExponential</li>
	<li>javascript.builtins.Number.toFixed</li>
	<li>javascript.builtins.Number.toLocaleString</li>
	<li>javascript.builtins.Number.toPrecision</li>
	<li>javascript.builtins.Number.toString</li>
	<li>javascript.builtins.Number.valueOf</li>
</ul>
</details>

<details>
<summary>7: Object (#5427) ✅</summary>
<ul>
	<li>javascript.builtins.Object</li>
	<li>javascript.builtins.Object.constructor</li>
	<li>javascript.builtins.Object.defineGetter</li>
	<li>javascript.builtins.Object.defineSetter</li>
	<li>javascript.builtins.Object.hasOwnProperty</li>
	<li>javascript.builtins.Object.isPrototypeOf</li>
	<li>javascript.builtins.Object.lookupGetter</li>
	<li>javascript.builtins.Object.lookupSetter</li>
	<li>javascript.builtins.Object.propertyIsEnumerable</li>
	<li>javascript.builtins.Object.proto</li>
	<li>javascript.builtins.Object.prototype</li>
	<li>javascript.builtins.Object.toLocaleString</li>
	<li>javascript.builtins.Object.toString</li>
	<li>javascript.builtins.Object.valueOf</li>
</ul>
</details>

<details>
<summary>8: RegExp (#5429) ✅</summary>
<ul>
	<li>javascript.builtins.RegExp</li>
	<li>javascript.builtins.RegExp.compile</li>
	<li>javascript.builtins.RegExp.exec</li>
	<li>javascript.builtins.RegExp.flags</li>
	<li>javascript.builtins.RegExp.global</li>
	<li>javascript.builtins.RegExp.global.prototype_accessor</li>
	<li>javascript.builtins.RegExp.ignoreCase</li>
	<li>javascript.builtins.RegExp.ignoreCase.prototype_accessor</li>
	<li>javascript.builtins.RegExp.input</li>
	<li>javascript.builtins.RegExp.lastIndex</li>
	<li>javascript.builtins.RegExp.lastMatch</li>
	<li>javascript.builtins.RegExp.lastParen</li>
	<li>javascript.builtins.RegExp.leftContext</li>
	<li>javascript.builtins.RegExp.multiline</li>
	<li>javascript.builtins.RegExp.multiline.prototype_accessor</li>
	<li>javascript.builtins.RegExp.n</li>
	<li>javascript.builtins.RegExp.prototype</li>
	<li>javascript.builtins.RegExp.rightContext</li>
	<li>javascript.builtins.RegExp.source</li>
	<li>javascript.builtins.RegExp.source.empty_regex_string</li>
	<li>javascript.builtins.RegExp.source.escaping</li>
	<li>javascript.builtins.RegExp.source.prototype_accessor</li>
	<li>javascript.builtins.RegExp.sticky.anchored_sticky_flag</li>
	<li>javascript.builtins.RegExp.sticky.prototype_accessor</li>
	<li>javascript.builtins.RegExp.test</li>
	<li>javascript.builtins.RegExp.toString</li>
	<li>javascript.builtins.RegExp.toString.escaping</li>
	<li>javascript.builtins.RegExp.@@match</li>
	<li>javascript.builtins.RegExp.@@replace</li>
	<li>javascript.builtins.RegExp.@@search</li>
	<li>javascript.builtins.RegExp.@@species</li>
	<li>javascript.builtins.RegExp.@@split</li>
</ul>
</details>

<details>
<summary>9: String (#5431) ✅</summary>
<ul>
	<li>javascript.builtins.String</li>
	<li>javascript.builtins.String.anchor</li>
	<li>javascript.builtins.String.big</li>
	<li>javascript.builtins.String.blink</li>
	<li>javascript.builtins.String.bold</li>
	<li>javascript.builtins.String.charAt</li>
	<li>javascript.builtins.String.charCodeAt</li>
	<li>javascript.builtins.String.concat</li>
	<li>javascript.builtins.String.fixed</li>
	<li>javascript.builtins.String.fontcolor</li>
	<li>javascript.builtins.String.fontsize</li>
	<li>javascript.builtins.String.fromCharCode</li>
	<li>javascript.builtins.String.indexOf</li>
	<li>javascript.builtins.String.italics</li>
	<li>javascript.builtins.String.lastIndexOf</li>
	<li>javascript.builtins.String.length</li>
	<li>javascript.builtins.String.link</li>
	<li>javascript.builtins.String.localeCompare</li>
	<li>javascript.builtins.String.match</li>
	<li>javascript.builtins.String.prototype</li>
	<li>javascript.builtins.String.replace</li>
	<li>javascript.builtins.String.search</li>
	<li>javascript.builtins.String.slice</li>
	<li>javascript.builtins.String.small</li>
	<li>javascript.builtins.String.split</li>
	<li>javascript.builtins.String.strike</li>
	<li>javascript.builtins.String.sub</li>
	<li>javascript.builtins.String.substr</li>
	<li>javascript.builtins.String.substring</li>
	<li>javascript.builtins.String.sup</li>
	<li>javascript.builtins.String.toLocaleLowerCase</li>
	<li>javascript.builtins.String.toLocaleLowerCase.locale</li>
	<li>javascript.builtins.String.toLocaleUpperCase</li>
	<li>javascript.builtins.String.toLocaleUpperCase.locale</li>
	<li>javascript.builtins.String.toLowerCase</li>
	<li>javascript.builtins.String.toString</li>
	<li>javascript.builtins.String.toUpperCase</li>
	<li>javascript.builtins.String.unicode_code_point_escapes</li>
	<li>javascript.builtins.String.valueOf</li>
</ul>
</details>

<details>
<summary>10: Misc. Builtins (#5433) ✅</summary>
<ul>
	<li>javascript.builtins.Atomics</li>
	<li>javascript.builtins.Atomics.add</li>
	<li>javascript.builtins.Atomics.and</li>
	<li>javascript.builtins.Atomics.compareExchange</li>
	<li>javascript.builtins.Atomics.exchange</li>
	<li>javascript.builtins.Atomics.isLockFree</li>
	<li>javascript.builtins.Atomics.load</li>
	<li>javascript.builtins.Atomics.notify (as wake)</li>
	<li>javascript.builtins.Atomics.or</li>
	<li>javascript.builtins.Atomics.store</li>
	<li>javascript.builtins.Atomics.sub</li>
	<li>javascript.builtins.Atomics.wait</li>
	<li>javascript.builtins.Atomics.xor</li>
	<li>javascript.builtins.Boolean</li>
	<li>javascript.builtins.Boolean.prototype</li>
	<li>javascript.builtins.Boolean.toString</li>
	<li>javascript.builtins.Boolean.valueOf</li>
	<li>javascript.builtins.DataView.buffer.sharedarraybuffer_support</li>
	<li>javascript.builtins.DataView.new_required</li>
	<li>javascript.builtins.DataView.sharedarraybuffer_support</li>
	<li>javascript.builtins.Intl.DateTimeFormat.hourCycle</li>
	<li>javascript.builtins.Intl.NumberFormat.formatToParts</li>
	<li>javascript.builtins.JSON.json_superset</li>
	<li>javascript.builtins.Map.@@iterator</li>
	<li>javascript.builtins.Promise.allSettled</li>
	<li>javascript.builtins.Proxy.handler.isExtensible</li>
	<li>javascript.builtins.Proxy.handler.setPrototypeOf</li>
	<li>javascript.builtins.Set.@@iterator</li>
	<li>javascript.builtins.Symbol.hasInstance</li>
	<li>javascript.builtins.Symbol.isConcatSpreadable</li>
	<li>javascript.builtins.Symbol.match</li>
	<li>javascript.builtins.Symbol.replace</li>
	<li>javascript.builtins.Symbol.search</li>
	<li>javascript.builtins.Symbol.split</li>
	<li>javascript.builtins.Symbol.toPrimitive</li>
	<li>javascript.builtins.Symbol.toStringTag</li>
	<li>javascript.builtins.Symbol.@@toPrimitive</li>
</ul>
</details>

<details>
<summary>11: Top-Level Builtins (#5478) ✅</summary>
<ul>
	<li>javascript.builtins.Infinity</li>
	<li>javascript.builtins.NaN</li>
	<li>javascript.builtins.decodeURI</li>
	<li>javascript.builtins.decodeURIComponent</li>
	<li>javascript.builtins.encodeURI</li>
	<li>javascript.builtins.encodeURIComponent</li>
	<li>javascript.builtins.escape</li>
	<li>javascript.builtins.eval</li>
	<li>javascript.builtins.isFinite</li>
	<li>javascript.builtins.isNaN</li>
	<li>javascript.builtins.null</li>
	<li>javascript.builtins.parseFloat</li>
	<li>javascript.builtins.parseInt</li>
	<li>javascript.builtins.undefined</li>
	<li>javascript.builtins.unescape</li>
</ul>
</details>

<details>
<summary>12: Grammar (#5479) ✅</summary>
<ul>
	<li>javascript.grammar.array_literals</li>
	<li>javascript.grammar.boolean_literals</li>
	<li>javascript.grammar.decimal_numeric_literals</li>
	<li>javascript.grammar.hashbang_comments</li>
	<li>javascript.grammar.hexadecimal_escape_sequences</li>
	<li>javascript.grammar.hexadecimal_numeric_literals</li>
	<li>javascript.grammar.null_literal</li>
	<li>javascript.grammar.numeric_separators</li>
	<li>javascript.grammar.regular_expression_literals</li>
	<li>javascript.grammar.string_literals</li>
	<li>javascript.grammar.unicode_escape_sequences</li>
	<li>javascript.grammar.trailing_commas</li>
	<li>javascript.grammar.trailing_commas.trailing_commas_in_object_literals</li>
</ul>
</details>

<details>
<summary>13: Operators (#5480) ✅</summary>
<ul>
	<li>javascript.operators.arithmetic.addition</li>
	<li>javascript.operators.arithmetic.decrement</li>
	<li>javascript.operators.arithmetic.division</li>
	<li>javascript.operators.arithmetic.increment</li>
	<li>javascript.operators.arithmetic.multiplication</li>
	<li>javascript.operators.arithmetic.remainder</li>
	<li>javascript.operators.arithmetic.subtraction</li>
	<li>javascript.operators.arithmetic.unary_negation</li>
	<li>javascript.operators.arithmetic.unary_plus</li>
	<li>javascript.operators.assignment.addition</li>
	<li>javascript.operators.assignment.bitwise_and</li>
	<li>javascript.operators.assignment.bitwise_or</li>
	<li>javascript.operators.assignment.bitwise_xor</li>
	<li>javascript.operators.assignment.division</li>
	<li>javascript.operators.assignment.left_shift</li>
	<li>javascript.operators.assignment.multiplication</li>
	<li>javascript.operators.assignment.remainder</li>
	<li>javascript.operators.assignment.right_shift</li>
	<li>javascript.operators.assignment.simple</li>
	<li>javascript.operators.assignment.subtraction</li>
	<li>javascript.operators.assignment.unsigned_right_shift</li>
	<li>javascript.operators.bitwise.and</li>
	<li>javascript.operators.bitwise.left_shift</li>
	<li>javascript.operators.bitwise.not</li>
	<li>javascript.operators.bitwise.or</li>
	<li>javascript.operators.bitwise.right_shift</li>
	<li>javascript.operators.bitwise.unsigned_right_shift</li>
	<li>javascript.operators.bitwise.xor</li>
	<li>javascript.operators.class</li>
	<li>javascript.operators.comma</li>
	<li>javascript.operators.comparison.equality</li>
	<li>javascript.operators.comparison.inequality</li>
	<li>javascript.operators.comparison.identity</li>
	<li>javascript.operators.comparison.non_identity</li>
	<li>javascript.operators.comparison.greater_than</li>
	<li>javascript.operators.comparison.greater_than_or_equal</li>
	<li>javascript.operators.comparison.less_than</li>
	<li>javascript.operators.comparison.less_than_or_equal</li>
	<li>javascript.operators.conditional</li>
	<li>javascript.operators.delete</li>
	<li>javascript.operators.function</li>
	<li>javascript.operators.function_star.trailing_comma</li>
	<li>javascript.operators.grouping</li>
	<li>javascript.operators.in</li>
	<li>javascript.operators.instanceof</li>
	<li>javascript.operators.logical.and</li>
	<li>javascript.operators.logical.or</li>
	<li>javascript.operators.logical.not</li>
	<li>javascript.operators.new</li>
	<li>javascript.operators.new_target</li>
	<li>javascript.operators.optional_chaining</li>
	<li>javascript.operators.property_accessors</li>
	<li>javascript.operators.spread.spread_in_destructuring</li>
	<li>javascript.operators.super</li>
	<li>javascript.operators.this</li>
	<li>javascript.operators.typeof</li>
	<li>javascript.operators.void</li>
</ul>
</details>

<details>
<summary>14: Statements (#5481) ✅</summary>
<ul>
	<li>javascript.statements.block</li>
	<li>javascript.statements.break</li>
	<li>javascript.statements.continue</li>
	<li>javascript.statements.debugger</li>
	<li>javascript.statements.default.switch</li>
	<li>javascript.statements.do_while</li>
	<li>javascript.statements.empty</li>
	<li>javascript.statements.for</li>
	<li>javascript.statements.for_in</li>
	<li>javascript.statements.for_of.async_iterators</li>
	<li>javascript.statements.for_of.closing_iterators</li>
	<li>javascript.statements.function</li>
	<li>javascript.statements.function.trailing_comma_in_parameters</li>
	<li>javascript.statements.generator_function.IteratorResult_object</li>
	<li>javascript.statements.generator_function.trailing_comma_in_parameters</li>
	<li>javascript.statements.if_else</li>
	<li>javascript.statements.label</li>
	<li>javascript.statements.return</li>
	<li>javascript.statements.switch</li>
	<li>javascript.statements.throw</li>
	<li>javascript.statements.try_catch</li>
	<li>javascript.statements.var</li>
	<li>javascript.statements.while</li>
	<li>javascript.statements.with</li>
</ul>
</details>

15: #5494
16: #5497

**Comment by vlkn79** [comment]
> 
> 
> Part of #4771. Below are JavaScript features that are set to `true` or `null` for Safari (macOS edition). The goal of this issue is to find the actual version numbers these features were implemented, if at all.
> Total amount of features: 436
> 
> _Note: Until Chrome 27, Chrome used the same engine as Safari (WebKit). As a result, you may be able to add Safari data for any features implemented in Chrome 1-27 by comparing the WebKit engine versions described in [browsers/chrome.json](https://github.com/mdn/browser-compat-data/blob/master/browsers/chrome.json) and [browser/safari.json](https://github.com/mdn/browser-compat-data/blob/master/browsers/safari.json). Additionally, Chrome 1 used a WebKit version just before Safari 4, so any features implemented in Safari 1-3.2 may be set to Chrome 1. Chrome 27 and Safari 6 were the last versions that maintained parity before Chrome forked WebKit into the Blink engine._
> 
> _Another note: I have VMs for Safari 1 through 7 should anyone interested in helping need copies. Please [email me](https://github.com/vinyldarkscratch) with the subject “MDN Safari VMs” and I’ll send a download link! For Safari 8+, we recommend a program like SauceLabs or BrowserStack; alternatively I can configure VMs per request._
> 1: Array (#5249) white_check_mark
> 2: Date (#5302) white_check_mark
> 3: Error (#5315) white_check_mark
> 4: Function
> 5: Math
> 6: Number
> 7: Object
> 8: RegExp
> 9: String
> 10: Misc. Builtins
> 11: Top-Level Builtins
> 12: Grammar
> 13: Operators
> 14: Statements

**Comment by vinyldarkscratch** [comment]
Just got a few more remaining, and then this is done!

```
✖ javascript/builtins/Array.json
  Real values – 1 error:
  → javascript.builtins.Array.sort.stable - safari no longer accepts null as a value
✖ javascript/builtins/Intl.json
  Real values – 3 errors:
  → javascript.builtins.Intl.Collator.caseFirst - safari no longer accepts true as a value
  → javascript.builtins.Intl.DateTimeFormat.dateStyle - safari no longer accepts null as a value
  → javascript.builtins.Intl.DateTimeFormat.timeStyle - safari no longer accepts null as a value
```

The `javascript.builtins.Array.sort.stable`, I have no clue how to test and determine the proper version number, primarily as I've been seeing signs that it may be implemented in Safari versions where Chrome was built off of WebKit, but Chrome had no signs of implementation in those editions.  I would love to get some help on that one!
